test/data/test_mapper.py

```diff
@@ -38,6 +39,23 @@ def test_categorical_tensor_mapper():
     pd.testing.assert_series_equal(out, pd.Series(['A', 'B', None, None, 'B']))
 
 
+def test_multicategorical_tensor_mapper():
```

Comment:
Can you also add a test for `None` handling?

Reply:
Sure